ArjunaJTS/interop/glassfish/ejb_operations.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+source init.sh
+set +e
+
+export PATH=$GLASSFISH/bin:$PATH
+[ -d "$JBOSS_HOME"  ] || fatal "file not found: $JBOSS_HOME"
+[ -f "$GLASSFISH/bin/asadmin"  ] || fatal "asadmin not found"
+
+WF_DEPLOY_DIR=$JBOSS_HOME/standalone/deployments
+
+usage() {
+  echo -e "$1: Usage: $0 [-a <gf|gf2|wf>] [-f <archive>] [-t <gfgf|gfwf|wfgf >] [-t <haltbefore|haltafter>]" 1>&2;
+  echo -e "examples:"
+  echo -e "\t$0 -t wfgf # send a message from wf to gf"
```

Comment:
may I ask you for a final cosmetic change here in comments? `send a message from wildfly to glassfish` and similar on line below?